.gitignore

```diff
@@ -253,6 +253,10 @@ ui_*.h
 *.jsc
 Makefile*
 *build-*
+!build-aux
```

Comment:
The stuff under `windows_compat` makes sense to me, but it's not usual to need to commit build files; what is this `build-aux` directory and should it really be included?

Reply:
I see; so it's not build products, but things needed to build.  Can it go inside the windows_compat folder?

Reply:
It is inside `windows_compat/gnulib`. I assume it is okay there? It is placed there by the `gnulib-tool` which is used to setup `gnulib`, I wouldn't want to change it because this would require manually editing the generated `configure` script, which would be hard to keep track of.

Reply:
Oh, I see.  OK, just change the exclude here in .gitignore to give a path that's inside windows_compat, then.  My goal is for the .gitignore to be more self-documenting, so that it's clear that this is something excluded because of the Windows build, so I don't get confused by it later on.  :->

Reply:
Okay fair enough! Will do.